src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import Button from "../Button";
+import "./Navbar.css";
+
+const Navbar = () => {
+  const [navItem, setNavItem] = useState("Home");
```

Comment:
`selectedNavItem`

Reply:
https://codedamn.com/news/javascript/the-art-of-naming-variables-in-javascript-tips-and-best-practices

Reply:
ok